frontend/src/components/ConfirmationModal.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react'
+import Button from './Button'
+import styled from 'styled-components'
+import { useHistory } from 'react-router-dom'
+import { PATHS } from '../routes/Routes'
+import { JsxElement } from 'typescript'
+
+const OverlayContainer = styled.div<{ overlayBackgroundColor: string }>`
+  position: fixed;
+  min-height: 100vh;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: ${(props) => (props.overlayBackgroundColor ? props.overlayBackgroundColor : 'rgba(0, 0, 0, 0.3)')};
+  z-index: 999;
+`
+
+const MainContainer = styled.div`
+  position: fixed;
+  background-color: #fff;
+  width: 90vw;
+  border-radius: 15px;
+  margin: auto;
+  padding: 15px;
+  box-shadow: 0 2px 5px 1px #888888;
+  z-index: 1000;
+  display: flex;
+  flex-direction: column;
+`
+
+const TitleText = styled.h1`
+  font-family: Lato;
+  font-size: 22px;
+  font-weight: 400;
+  text-align: center;
+  font-style: normal;
+`
+
+const DescriptionText = styled.p`
+  font-family: Lato;
+  font-size: 14spx;
+  font-weight: 40;
+  text-align: center;
+  font-style: normal;
+`
+
+const ButtonContainer = styled.div`
+  display: flex;
+  justify-content: space-evenly;
+  width: 100%;
+  margin-top: 10px;
+`
+
+const Conflictlink = styled.a`
+  font-family: Lato;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 16.8px;
```

Comment:
is there a reason why this value is 16.8?

Reply:
Following Figma styling.